fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -0,0 +1,30 @@
+The KUKSA Companion App demonstrates the capabilities of the Android SDK to connect to an
+Android Automotive vehicle with an integrated databroker, read data and interact with the
```

Comment:
Could we mention the Coves VSS here? Maybe a short sentence behind the Databroker. Because in the next paragraph it is popping up a bit out of the blue.